config.py

```diff
@@ -2,13 +2,12 @@
 from dotenv import load_dotenv
 
 basedir = path.abspath(path.dirname(__file__))
-#load_dotenv(path.join(basedir, '.env'))
```

Comment:
notice you changed this back

Reply:
yes, I found a more elegant way to address this by passing an `or` statement when loading the .env variable.

Reply:
ok thats fine